docs/art-direction.md

```diff
@@ -0,0 +1,21 @@
+[Back home](/)
+
+# Use case: Art Direction
+
+How to deal with multiple `<source>` with `mix/max-width` media queries (Art Direction)?
+
+If you have some code like this:
+
+```html
+<picture>
+  <source media="(min-width: 800px)" srcset="…" sizes="…">
+  <img srcset="…" sizes="…" alt="…">
+</picture>
+```
+
+You will have to run the script twice, with (at least) these parameters, to get widths for both `srcset`s:
+
+```shell
+npx responsive-image-widths --max-viewport 799
```

Comment:
this commands can’t currently work without `--verbose` or `--destFile`

Reply:
Indeed…